nextcloud_scan.py

```diff
@@ -66,42 +66,58 @@ def getResultJson(self):
         result = self.doScan()
 
         result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
-        result['hardeningsMissing'] = sum( 1 for isHardened in result['hardenings'].values() if isHardened == False )
-        result['headersMissing'] = sum( 1 for headerPresent in result['setup']['headers'].values() if headerPresent == False )
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
 
-        return json.dumps(result, separators=(',',':'))
+        return json.dumps(result, separators=(',', ':'))
+
+    def getFormattedResultJson(self):
+        result = self.doScan()
+
+        result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
+
+        # Hier erfolgt die formatierte Ausgabe mit Einrückungen
+        return json.dumps(result, indent=4, separators=(',', ':'))
 
 def main():
     parser = argparse.ArgumentParser(
-                                    description='Run a Nextcloud security scan at scan.nextcloud.org',
-                                    )
+        description='Run a Nextcloud security scan at scan.nextcloud.org',
+    )
     parser.add_argument('hostname',
                         help='Publicly-available hostname of the Nextcloud instance to scan'
-                       )
+                        )
     parser.add_argument('-u', '--uri',
                         required=False,
                         default='/',
                         help='Base URI of the Nextcloud instance (default: "%(default)s")'
-                       )
+                        )
     parser.add_argument('-m', '--minutes',
                         required=False,
                         default=1440,
                         type=int,
                         help='Number of minutes after which to initiate a re-scan (default: %(default)s)'
-                       )
+                        )
     parser.add_argument('-D', '--debug',
                         required=False,
                         default=False,
                         action='store_true'
-                       )
+                        )
+    parser.add_argument('-V', '--verbose',
```

Comment:
`-v` (lowercase) is the more common verbose flag letter. Could you change to that?